spec_creation/Validate_spec_before_upload.ipynb

```diff
@@ -241,40 +233,80 @@
     "                        lonlat_swap(wpc), popup=\"%d\" % i,\n",
     "                        icon=fof.DivIcon(class_name='leaflet-div-icon')).add_to(curr_map)\n",
     "    \n",
-    "    latlng_route_coords = [[lonlat_swap(rc) for rc in coords[\"geometry\"][\"coordinates\"]] for coords in trip[\"route_coords\"]]\n",
-    "    for ll in latlng_route_coords:\n",
-    "        folium.PolyLine(ll, popup=\"%s: %s\" % (trip[\"mode\"], trip[\"name\"])).add_to(curr_map)\n",
-    "        \n",
-    "    for i, c in enumerate(latlng_route_coords):\n",
-    "        folium.CircleMarker(c, radius=5, popup=\"%d: %s\" % (i, c)).add_to(curr_map)\n",
-    "        \n",
-    "    curr_map.fit_bounds(ful.get_bounds(trip[\"route_coords\"][0][\"geometry\"][\"coordinates\"], lonlat=True))\n",
+    "    for rc in trip[\"route_coords\"]:\n",
+    "        print(\"Found %d coordinates for the route\" % (len(rc[\"geometry\"][\"coordinates\"])))\n",
```

Comment:
This looks fine, but it is not really the number of coordinates for the route. It is the number of reroutes, right?
If you were able to run the original notebook, you would see that the number of coordinates would be in the hundreds.